### DISCUSSION TITLE
Airflow log table many records with execution_date as NULL

### DISCUSSION THREAD

**Discussion by bhanu-prakash3** [apache/airflow#22423] [Q&A]
I'm using Airflow version - 2.2.3. The Airflow Database used is Postgres with a storage space of 6GB. The log table alone is occupying 1.9GB of the space. I ran this query to check at which date most number of logs are recorded.

`select count(*), execution_date from log group by execution_date `

![Screenshot from 2022-03-18 15-49-04](https://user-images.githubusercontent.com/44705470/159476617-b61494e0-7c1b-45e7-91b8-2019fadd5c53.png)

Observed that many logs have execution_date as NULL. I'm not sure on which date 1765 logs are recorded.

I know that we can purge these records https://github.com/apache/airflow/discussions/13815

But everytime deleting the records from log table takes too much time. Is there any way to stop logging into log table?

Can anyone advise why logs are stored with execution_date as NULL.

**Comment by potiuk** [comment]
It's wrong field you are looking at. The execution_date is just parameter (and only set for task-related logs). There are a number of other logs which are not task related. 

You should use dttm field to see when the log was created insteade

**Reply by bhanu-prakash3** [reply]
Thanks for the reply, yes dttm field shows when the log was created. There are 12 active running DAGS in my Airflow setup. For each day more than 200 logs are being recorded in the log table. Is there any way to stop logging?

**Reply by potiuk** [reply]
no idea to be honest - but check the configuration
